build/MPack.targets

```diff
@@ -1,12 +1,37 @@
 ﻿<Project>
   <PropertyGroup>
-    <PackageDependsOn Condition="'$(OS)'=='Windows_NT'">$(PackageDependsOn);GenerateMPack</PackageDependsOn>
```

Comment:
We're purposefully turning off mpack generation for this branch because it's for VS windows only and doesn't have all the product bits required to ship an mpack.

Reply:
Consider adding a boolean property like `BuildMPack` that defaults to `false` in this branch instead of deleting code. This makes merging between branches easier. Also, you can share this setting in multiple places.
 i.e. you also need to disable `GetMPackArtifactInfo`